tests/AerospikeCacheTest.php

```diff
@@ -0,0 +1,150 @@
+<?php
+declare(strict_types=1);
+
+namespace Lmc\AerospikeCache;
+
+use PHPUnit\Framework\TestCase;
+
+class AerospikeCacheTest extends TestCase
+{
+    /**
+     * @dataProvider doHaveProvider
+     */
+    public function testDoHave(int $statusCode, bool $expectedValue): void
+    {
+        $aerospikeMock = $this->createMock(\Aerospike::class);
+        $aerospikeCache = new AerospikeCache($aerospikeMock, 'test', 'cache');
+
+        $aerospikeKey = ['ns' => 'test', 'set' => 'cache', 'key' => 'foo'];
+        $aerospikeMock->expects($this->once())
+            ->method('initKey')
+            ->willReturn($aerospikeKey);
+
+        $aerospikeMock->expects($this->once())
+            ->method('get')
+            ->with($aerospikeKey)
+            ->willReturn($statusCode);
+
+        $hasItem = $aerospikeCache->hasItem('foo');
+
+        $this->assertEquals($expectedValue, $hasItem);
+    }
+
+    public function doHaveProvider(): array
+    {
+        return [
+            [\Aerospike::OK, true],
+            [\Aerospike::ERR_RECORD_NOT_FOUND, false],
+        ];
+    }
+
+    /**
+     * @dataProvider doSaveProvider
+     */
+    public function testDoSave(int $statusCode, bool $expectedResult): void
+    {
+        $aerospikeMock = $this->createMock(\Aerospike::class);
+        $aerospikeCache = new AerospikeCache($aerospikeMock, 'test', 'cache');
+
+        $aerospikeMock->method('put')
+                ->willReturn($statusCode);
+
+        $aerospikeMock->method('get')
+                ->willReturn(\Aerospike::OK);
+
+        $aerospikeKey = ['ns' => 'test', 'set' => 'cache', 'key' => 'foo'];
+        $aerospikeMock->method('initKey')
+                ->willReturn($aerospikeKey);
+
+        $cacheItem = $aerospikeCache->getItem('foo');
+        $result = $aerospikeCache->save($cacheItem);
+
+        $this->assertEquals($expectedResult, $result);
+    }
+
+    public function doSaveProvider(): array
+    {
+        return [
+            [\Aerospike::OK, true],
+            [\Aerospike::ERR_CLIENT, false],
+        ];
+    }
+
+    /**
+     * @dataProvider doClearWithEmptyNamespaceProvider
+     */
+    public function testDoClearWithEmptyNamespace(int $statusCode, bool $expectedResult): void
+    {
+        $aerospikeMock = $this->createMock(\Aerospike::class);
+        $aerospikeCache = new AerospikeCache($aerospikeMock, 'test', 'cache');
+
+        $aerospikeMock->method('truncate')->willReturn($statusCode);
```

Comment:
```suggestion
        $aerospikeMock->method('truncate')
            ->willReturn($statusCode);
```

For readability i suggest using one `->` per line. Also it is divided like this on different places in this file (line 49, 52, 56 etc.),  so it should be unified.

Reply:
ok done